coalib/output/Logging.py

```diff
@@ -4,25 +4,40 @@
 import logging
 import logging.config
 
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
 
-def configure_logging():
+
+def configure_logging(new_log_level=logging.INFO, incremental=False):
     """
     Configures the logging with hard coded dictionary.
     """
     import sys
 
     logging.config.dictConfig({
         'version': 1,
+        'incremental': incremental,
```

Comment:
We use levels a lot in testing and doctests.
I dont know exactly when we do this.
But basically when we configure_logging for the first time we set increment to false.
When we only want to change the level we set it to True.